### ISSUE TITLE
Difficulty in reproducing code. 

### ISSUE THREAD

**Issue by AKANKSHASINGH233** [NKU-HLT/PB-DSR#2]
Hello Sir,

I am facing trouble to access dataset can you help me with procedure and division of dataset as enlisted in the original paper

**Comment by AKANKSHASINGH233** [comment]
Hello sir,
Please help me with splitting dataset i am not able to find the same in this github repo

**Comment by AKANKSHASINGH233** [comment]
i have done data split and preprocessing as mentioned in the paper yet WER i am getting after fine-tuning on the UASpeech dataset using CTC loss only to get model R, i am getting WER as "VALID_best_wer": "99.778", it is not reducing. I cross-checked all steps still i am not getting why it happening.

**Comment by AKANKSHASINGH233** [comment]
Hi @wsywsywsywsywsy979 
please help here for the above mentioned issues.

**Comment by wsywsywsywsywsy979** [comment]
> i have done data split and preprocessing as mentioned in the paper yet WER i am getting after fine-tuning on the UASpeech dataset using CTC loss only to get model R, i am getting WER as "VALID_best_wer": "99.778", it is not reducing. I cross-checked all steps still i am not getting why it happening.

Could you please display the TensorBoard? Additionally, can you confirm that all settings align with the specifications outlined in our paper, including the learning rate, dictionary, and the correct pre-trained models that have been loaded?
